package.json

```diff
@@ -30,13 +30,15 @@
   },
   "homepage": "https://github.com/NinoMaj/carousel-virtualized#readme",
   "dependencies": {
+    "@types/lodash": "^4.14.116",
```

Comment:
`@types` should go to devDependencies

Reply:
You are right for privately used properties and methods like lodash, but I believe that for publicly exposed one, like react-window, we should leave them in deps. Do you agree?